utils/gen-snapshot-tests.js

```diff
@@ -29,7 +29,7 @@ const ignored = [
 
 function generateTestFile(root, examples, componentFileMap) {
     const lines = [
-        "// @flow",
+        // "// @flow",
```

Comment:
Ahhh - that should be fixed, I suspect!

Reply:
Yeah, I'll add it to the infra epic in JIRA.